assembly/src/main/jib/app/stream-registry-config.yaml

```diff
@@ -9,6 +9,10 @@ server:
 # Env where StreamRegistry application is running
 env: ${STREAM_REGISTRY_ENVIRONMENT:-local}
 
+# list (yaml syntax) of Jersey Filter Full Class Names loaded at runtime.
```

Comment:
provide an example of this (commented out) so that a person can uncomment

e.g. 
```
# uncomment this section with your specific Jersey filters if you want to add some customization that requires filters
```

Reply:
do you want me do add 
https://github.homeawaycorp.com/StreamingPlatform/ha-stream-registry/pull/223/files#diff-7b5a9cdf95acab871aadc42f7d7302d1

```/src/main/java/com/homeaway/streamplatform/extensions/filters/UUIDMDCResponseFilter.java```
and people can uncomment it.

Reply:
like

```
requestFilterClassNames:
       - com.homeaway.streamplatform.extensions.filters.UUIDMDCRequestFilter      
responseFilterClassNames:
       - com.homeaway.streamplatform.extensions.filters.UUIDMDCResponseFilter
```

Reply:
fixed.